plugins/org.eclipse.emf.mwe2.language/META-INF/MANIFEST.MF

```diff
@@ -11,11 +11,11 @@ Require-Bundle: org.eclipse.emf.ecore,
  org.objectweb.asm;bundle-version="[6.1.1,10.0.0)";resolution:=optional,
  org.eclipse.equinox.common;bundle-version="3.16.0",
  com.google.inject;bundle-version="7.0.0",
- org.eclipse.xtext;bundle-version="2.32.0",
- org.eclipse.xtext.common.types;bundle-version="2.32.0";visibility:=reexport,
+ org.eclipse.xtext;bundle-version="2.34.0",
```

Comment:
It feels like we should restrict all Xtext bundles to 2.34 and above to guide the P2 resolver. I'll do that after landing this MR.